--- psensor-0.4.4/Makefile	2013-02-08 14:25:54.022571318 +0200
+++ psensor-0.4.4/Makefile	2013-02-08 14:27:49.654581619 +0200
@@ -1,6 +1,8 @@
 # Released under GPLv2 license terms, read LICENSE file
 
 PREFIX=/usr
+LIB=lib
+LIBDIR=$(PREFIX)/$(LIB)
 
 CC=gcc
 
@@ -25,11 +27,11 @@
 OBJS=main.o graph.o psensor.o ui_sensorlist.o config.o color.o hdd.o ui_color.o
 
 # uncomments for compilation with nvidia thermal sensor support
-#LD_OPTS+= /usr/lib/libXNVCtrl.a
-#OBJS+= nvidia.o
-#CC_D+= -D NVIDIA_SUPPORT
+LD_OPTS+= $(LIBDIR)/libXNVCtrl.a
+OBJS+= nvidia.o
+CC_D+= -D NVIDIA_SUPPORT
 
-CC_OPTS=-Wall -O3 `pkg-config --cflags --libs ${PKG_CONFIG_LIBRARIES}` ${CC_D} 
+CC_OPTS=$(OPTFLAGS) `pkg-config --cflags --libs ${PKG_CONFIG_LIBRARIES}` ${CC_D} 
 
 ################### PSensor Server ######################
 
@@ -38,11 +40,11 @@
 LD_OPTS_SERVER= -lsensors
 
 # uncomments for compilation with nvidia thermal sensor support
-#OBJS_SERVER+= nvidia.o 
-#CC_D_SERVER+= -D NVIDIA_SUPPORT
-#LD_OPTS_SERVER+= -lX11 -lXext /usr/lib/libXNVCtrl.a
+OBJS_SERVER+= nvidia.o 
+CC_D_SERVER+= -D NVIDIA_SUPPORT
+LD_OPTS_SERVER+= -lX11 -lXext $(LIBDIR)/libXNVCtrl.a
 
-CC_OPTS_SERVER=-Wall -O3 ${CC_D_SERVER}
+CC_OPTS_SERVER=$(OPTFLAGS) ${CC_D_SERVER}
 
 all:psensor psensor-server
 
